docs/HTTPSCertificates.md

```diff
@@ -0,0 +1,46 @@
+# Managing Nginx HTTPS Certificates
+
+We use Nginx to help provide secure HTTP communication between clients and servers. Nginx achieves this through HTTPS (HTTP over TLS/SSL). We have two requirements from HTTPS. We require a client to be able to verify the identity of a server, and we then require the client and the server to be able to communicate securely. We achieve the first of these two requirements through HTTPS certificates.
+
+In this document we'll refer to the following terms.
+* Public key - a public cryptographic key.
+* Private key - the corresponding private cryptographic key.
+* Certificate signing request (CSR) - a public key with a service provider's information embedded in it.
+* Certificate Authority (CA) - a body that signs a CSR with the private key of a known certificate.
+* Certificate - a signed CSR.
```

Comment:
A signed public key, not a signed CSR.  Possibly it's more useful to say what the file is for (here it's used by the server to try to prove it's identity to the client)?